### ISSUE TITLE
"before all" hook: codeceptjs.beforeSuite:Cannot read property '0' of null

### ISSUE THREAD

**Issue by ninpop25** [Codeception/CodeceptJS#901]
#### I am trying to launch my test case on Internet Explorer on my client server on azure with **Window 10 Enterprise N**..for this I am starting the selenium standalone server automatically by using **start_server.js**, **stop_server.js** and **system_config.js**..I have installed latest selenium standalone with latest drivers of all browsers i.e (chrome,firefox and IE).

For IE I have manually placed **IEdriver version 2.53.1** in location AppData/Roaming/npm/node_modules/selenium-standalone/.selenium as with latest version of IE there is some issue and my it don't launch the IE when my tests starts.

I am using **codecept version 1.0.3**, npm version on my client server is 5.3.0 and Node version is 8.2.1

#### With only change in npm version(5.5.0) on my local machine and everything else is same as I described above now when I am trying to launch the IE browser by using this command (npm run test:multi_ie) on client server then I am getting below error,but on my local system with windows 7 I am able to launch the IE without any issue.

**Error When I run on client server.**

> C:\Tests_Debug>npm run test:multi_ie

> fvautomation@1.0.0 test:multi_ie C:\Tests_Debug
> codeceptjs run-multiple "ie:internet explorer"

null
[1.ie:internet explorer] CodeceptJS v1.0.3
[1.ie:internet explorer] Using test root "C:\Tests_Debug"

[1.ie:internet explorer] Fv anonymousShpntSbmt --
[1.ie:internet explorer]  × "before all" hook: codeceptjs.beforeSuite in 1666ms
[1.ie:internet explorer] TypeError: Cannot read property '0' of null

[1.ie:internet explorer] -- FAILURES:

  1) Fv anonymousShpntSbmt: "before all" hook: codeceptjs.beforeSuite:
     Cannot read property '0' of null

```bash
# paste output here
```

> Provide test source code if related

```js
// paste test
```

### Details

* CodeceptJS version:1.0.3
* NodeJS Version:8.2.1
* Operating System:Windows 10
* Protractor || WebDriverIO || Nightmare version (if related)
* Configuration file: Please find [link](https://gist.github.com/ninpop25/9e82c6240e867bd40d9d46fa9e094b25) 

```js
# paste suite config here
```
@reubenmiller : Please help me in this

**Comment by reubenmiller** [comment]
I would highly recommend updating to Nodejs 8.9.3, but I just did some research and it is a bug with npm 5.5 (see the issue on the [npm repo](https://github.com/npm/npm/issues/17858).

So the fix should be to update npm to 5.6.0. Though I still recommend updating nodejs to 8.9.3 as well.

Note you will problably have to remove your `node_modules` folder and reinstall the all of the dependencies using `npm install`.

**Comment by ninpop25** [comment]
@reubenmiller : I first tried to update Npm but its not getting updated.

I removed the node_modules folder and then by below steps I try to updated the Npm but its not getting updated..I also used **npm install** but again same npm not getting installed but with version 5.3.0

C:\Tests_Debug>**npm install npm@5.6.0**
npm WARN fvautomation@1.0.0 No description
npm WARN fvautomation@1.0.0 No repository field.
npm WARN The package mocha is included as both a dev and production dependency.

+ npm@5.6.0
updated 1 package in 32.734s

C:\Tests_Debug>npm version
{ fvautomation: '1.0.0',
  **npm: '5.3.0',**
  ares: '1.10.1-DEV',
  cldr: '31.0.1',
  http_parser: '2.7.0',
  icu: '59.1',
  modules: '57',
  node: '8.2.1',
  openssl: '1.0.2l',
  tz: '2017b',
  unicode: '9.0',
  uv: '1.13.1',
  v8: '5.8.283.41',
  zlib: '1.2.11' }

**Do I need to first update the nodejs to 8.9.3 and will this updating of nodejs effect any of my settings?**

**Comment by reubenmiller** [comment]
Google is your friend with these sorts of problems. You probably have to at least start your shell session again (always good to do incase any environment variables were updated).

It is worth your while checking the node release page and npm repos for information about these issues. I just saw a note about the new node 8.9.4 release which added npm 5.6.0 as a dependency. So try installing node 8.9.4.

And there is only one way to find out if any of your config will be affected ;)
